src/common/pf_lldp.c

```diff
@@ -719,26 +749,100 @@ int pf_lldp_get_peer_station_name (
    int loc_port_num,
    pf_lldp_station_name_t * p_station_name)
 {
-   bool is_received;
+   pf_lldp_chassis_id_t chassis_id;
+   pf_lldp_port_id_t port_id;
+   char * p;
+   bool found = false;
+
    const pf_port_t * p_port_data = pf_port_get_state (net, loc_port_num);
+   memset (p_station_name, 0, sizeof (*p_station_name));
 
    os_mutex_lock (net->lldp_mutex);
+   port_id = p_port_data->lldp.peer_info.port_id;
+   chassis_id = p_port_data->lldp.peer_info.chassis_id;
+   os_mutex_unlock (net->lldp_mutex);
+
+   if (port_id.is_valid)
+   {
+      p = strchr (port_id.string, '.');
+      if (p == NULL)
```

Comment:
```suggestion
      if (p != NULL)
```

Reply:
Oups.. Maybe this explains failing test. Done